src/components/LeftSidebar/ConversationsList/Conversation.vue

```diff
@@ -21,7 +21,8 @@
 			<ConversationIcon :item="item" :hide-favorite="false" :hide-call="false" />
 		</template>
 		<template #subname>
-			{{ conversationInformation }}
+			<!-- eslint-disable-next-line vue/no-v-html -->
+			<span v-html="conversationInformation" />
```

Comment:
No? 
The message | the LS 
-- | --
![image](https://github.com/user-attachments/assets/a3ec836f-f431-40d2-aa69-f1ed6ddbe114) | ![image](https://github.com/user-attachments/assets/cf6c28f4-d9e6-43c5-8853-5392ea9ff0bd)